ui/src/actions/remoteAccess.js

```diff
@@ -30,13 +31,12 @@ export function updateNickname(name) {
   };
 }
 
-export function requestControl(
-  control = true,
-  message = '',
-  name = '',
-  userInfo = {},
-) {
-  return () => sendRequestControl(control, message, name, userInfo);
+export function requestControl(message) {
+  return () => sendRequestControl(message);
```

Comment:
`message`, `name`, `userInfo` are not actually used in the back-end. I keep `message` for now, since the goal is to eventually show the message to the user in control.